posts/affine-group-scheme-hopf-algebra.md

```diff
@@ -0,0 +1,269 @@
+Schemes are therefore the algebraic analogue of manifolds.
+
+The polynomial equations appearing in the definition of schemes can be over any commutative ring,
+but for simplicity we will assume that they are over $\mathbb C$.
+
+The complex numbers $\mathbb C$ and its unit group $\mathbb C^\times$ both have analogues as schemes.
+They are in this context referred to as the **affine line** $\mathbb A^1$ and **multiplicative group** $\mathbb G_m$.
+Their products $\mathbb A^n$ and $\mathbb G_m^n$ are called the **$n$-dimensional affine space** and **$n$-dimensional algebraic torus** respectively.
+
+A special class of schemes, namely *affine schemes*,
+can be constructed using the *Spec functor*,
+taking a commutative ring $R$ to its *prime spectrum* $\mathrm{Spec} R$,
+which is an affine scheme.
+The previous sentence is a bit scary,
+but all we need to know is that if $R$ and $S$ are commutative rings,
+then morphisms $\mathrm{Spec}\ R \to \mathrm{Spec}\ S$
+exactly correspond to ring homomorphisms $S \to R$.
+
+$\mathbb A^n$ is in fact isomorphic to $\mathrm{Spec}\ \mathbb C[X_1, \dots, X_n]$,
+the spectrum of the polynomial ring in $n$ variables.
+Similarly, $\mathbb G_m^n$ is isomorphic to $\mathrm{Spec}\ \mathbb C[X_1^{\pm 1}, \dots, X_n^{\pm 1}]$,
+the spectrum of multivariate Laurent series in $n$ variables.
+The natural inclusion $\mathbb C[X_1, \dots, X_n] \hookrightarrow \mathbb C[X_1^{\pm 1}, \dots, X_n^{\pm 1}]$
+corresponds to an embedding $\mathbb G_m^n \hookrightarrow \mathbb A^n$ with dense image
+(picture this as $(\mathbb C^\times)^n$ sitting in $\mathbb C^n$).
+
+# Group schemes
+
+Recall the definition of a group. A *group structure* on a set $G$ consists of:
+1. An **identity** $e \in G$.
+2. A **multiplication** $(\cdot * \cdot) : G \times G \to G$.
+3. An **inverse** $(\cdot^{-1}) : G \to G$.
+
+respecting the following *group axioms*:
+1. For all $g \in G$, $e * g = g$ (and $g * e = g$).
+2. For all $g, h, k \in G$, $(g * h) * k = g * (h * k)$.
+3. For all $g \in G$, $g^{-1} * g = e$ (and $g * g^{-1} = e$).
+
+A group structure can be described purely in terms of maps between
+$G, G \times G$ and the one element set $\{*\}$,
+meaning that we can reinterpret groups in any category $\mathsf{C}$
+where $(\cdot \times \cdot)$ and $\{*\}$ make sense,
+i.e. in any category with a binary product $(\cdot \otimes \cdot)$ and terminal object $\mathbf{1}_C$.
+A **group object** $G \in \mathsf{C}$ then consists of:
+1. A **unit morphism** $\eta : \mathbf{1}_C \to G$.
+2. A **multiplication morphism** $\mu : G \otimes G \to G$.
+3. An **inverse morphism** $\iota : G \to G$.
+
+making the following diagrams commute:
+<span style="color:red">**TODO: Add diagrams**</span>
+
+As an exercise try to come up with the diagrams that define a *commutative* group object.
+
+Analogously to group homomorphisms, if $G$ and $H$ are group objects,
+then a morphism $f : G \to H$ is a **group homomorphism** if the following two diagrams commute:
```

Comment:
```suggestion
then a morphism $f : G \to H$ is a **group scheme homomorphism** if the following two diagrams commute:
```

Reply:
That suggestion makes no sense, we haven't defined group schemes by this point